tests/core/string/test_fuzzy_search.h

```diff
@@ -110,8 +109,8 @@ auto bench(String p_query, Vector<String> p_targets) {
 			auto start = std::chrono::high_resolution_clock::now();
 			get_top_result(p_query, p_targets);
 			auto end = std::chrono::high_resolution_clock::now();
-			auto duration = std::chrono::duration_cast<std::chrono::microseconds>(end - start).count();
-			timings.push_back(duration / 1000.0); // Convert to ms
+			auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count();
```

Comment:
I had intentionally changed this since `duration` here is an integer type and when I was running the benchmark the average would end up being how many results got rounded to 7 vs how many got rounded to 8, so more precision felt correct. Is there a way to get a fractional duration here or some other reason/way to keep it ms?